vue-app/src/components/ProjectProfile.vue

```diff
@@ -30,30 +30,19 @@
         </div>
       </div>
       <div class="mobile mb2">
-        <div class="input-button" v-if="hasContributeBtn() && !inCart">
-          <img class="token-icon" height="24px" src="@/assets/dai.svg" />
-          <input
-            v-model="contributionAmount"
-            class="input"
-            name="contributionAmount"
-            placeholder="5"
-            autocomplete="on"
-            onfocus="this.value=''"
-          />
-          <input
-            type="submit"
-            class="donate-btn"
-            :disabled="!canContribute()"
-            @click="contribute()"
-            value="Add to cart"
-          />
-        </div>
-        <div class="input-button" v-if="hasContributeBtn() && inCart">
-          <button class="donate-btn-full">
-            <span>In cart 🎉</span>
-          </button>
-        </div>
+        <add-to-cart-button
+          v-if="shouldShowCartInput && hasContributeBtn()"
+          :project="project"
+        />
         <claim-button :project="project" />
+        <p
+          v-if="
+            $store.getters.hasUserContributed &&
+            !$store.getters.canUserReallocate
+          "
+        >
+          ✔️ You have contributed to this project!
```

Comment:
I like this feature! Only concern here - does this lead to potential bribery issues by providing a way someone can "prove" they contributed to a project? Definitely better than someone being able to prove this on chain but curious to hear what @auryn-macmillan thinks of adding this to the UI.